controllers/rack.go

```diff
@@ -620,6 +620,12 @@ func (r *SingleClusterReconciler) scaleDownRack(
 		}
 	}
 
+	// set migrate-fill-delay to 0 across all nodes of cluster to scale down fast
+	// temporarily add the pod to be deleted in ignorablePods slice to avoid setting migrate-fill-delay to pod when pod is not ready
```

Comment:
Have we faced any scenario that pod is not ready at this point of time? Because we are reducing the replicas after this step.

Reply:
This is only for the case when the pod which we are going to delete is not in running state. In that case, we skip the `waitForMultipleNodesSafeStopReady` check and then reduce the replica